### DISCUSSION TITLE
Chrome version help

### DISCUSSION THREAD

**Discussion by fes-google** [3kh0/ext-remover#742] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Yo, disregard the name, I'm using a school less monitored account to do this.

I need extension remover/blockers for the newer version of the chromebooks
(i got Version 115.0.5790.182 (Official Build (64-bit)) )

**Comment by AshtonDavies** [comment]
What blocker? (Lightspeed, Securly, etc)

**Reply by fes-google** [reply]
Lightspeed Filter Agent.
Seems most popular in my district.
They even have a location tracker.

**Reply by fes-google** [reply]
Coding4Hours
which is?

**Reply by S-PScripts** [reply]
> Coding4Hours which is?

https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/SOT%20Exploit%20-%20One%20Tab
